content/modules/webdav/pages/configuration.adoc

```diff
@@ -177,6 +177,32 @@ server.ssl.key-alias = localhost
 server.port = 8443
 ----
 
+=== Install trusted certificate
+
+Advanced use cases like opening WebDav files in LibreOffice or Firefox may not work with self-signed certificates. Trusted
```

Comment:
Don't think 'Advanced' should be here - it's not an advanced case itself, but more complex solution. And it works in LO before 7.3.
So maybe this will be better: `"Some use cases like opening WebDAV files in some version of LibreOffice or Firefox..." `